scripts/analyze_all_repos_github_actions.py

```
@@ -0,0 +1,434 @@
+    failed_runs: int = 0
+    workflow_breakdown: dict[str, float] = field(default_factory=dict)
+
+
+# GitHub Actions pricing (Linux)
+PRICING_LINUX = 0.008  # $0.008 per minute
+
+
+class MultiRepoAnalyzer:
+    """Analyze GitHub Actions usage across multiple repositories."""
+
+    def __init__(self, token: str):
```

Ruff (0.14.13)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)